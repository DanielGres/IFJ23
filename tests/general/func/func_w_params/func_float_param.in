// DOBRE
func need_tabak(_ input: Double) {
    write("go to trafika ", input);
}

need_tabak(631.4615846816865)