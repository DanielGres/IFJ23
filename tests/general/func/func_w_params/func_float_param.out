go to trafika 631.4615846816865